### ISSUE TITLE
rav1e not found, seems unrelated to #1910

### ISSUE THREAD

**Issue by 13steinj** [m-ab-s/media-autobuild_suite#1954]
Building for a 64-bit system, with most libraries enabled, I get an issue missing rav1e when building ffmpeg now. This seems to be new
``` bash
19:12:22 ├ Running configure...
Likely error (tail of the failed operation logfile):
CXXFLAGS: -mthreads -mtune=generic -O2 -pipe
LDFLAGS: -pipe -static-libgcc -static-libstdc++ -L/local64/lib -L/mingw64/lib
../configure --prefix=/local64 --bindir=/local64/bin-video --pkg-config=pkgconf --pkg-config-flags=--keep-system-libs --static --cc=ccache gcc --cxx=ccache g++ --ld=ccache g++ --disable-autodetect --enable-amf --enable-bzlib --enable-cuda --enable-cuvid --enable-d3d11va --enable-dxva2 --enable-iconv --enable-lzma --enable-nvenc --enable-zlib --enable-sdl2 --enable-ffnvcodec --enable-nvdec --enable-cuda-llvm --enable-libmp3lame --enable-libopus --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-libdav1d --enable-libaom --disable-debug --enable-libfdk-aac --enable-fontconfig --enable-libass --enable-libbluray --enable-libfreetype --enable-libmfx --enable-libmysofa --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-amrwbenc --enable-libwebp --enable-libxml2 --enable-libzimg --enable-libshine --enable-gpl --enable-avisynth --enable-libxvid --enable-libopenmpt --enable-version3 --enable-librav1e --enable-libsrt --enable-libgsm --enable-libvmaf --enable-libsvtav1 --enable-chromaprint --enable-decklink --enable-frei0r --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libflite --enable-libfribidi --enable-libgme --enable-libilbc --enable-libsvthevc --enable-libsvtvp9 --enable-libkvazaar --enable-libmodplug --enable-librtmp --enable-librubberband --enable-libtesseract --enable-libxavs --enable-libzmq --enable-libzvbi --enable-openal --enable-libcodec2 --enable-ladspa --enable-libglslang --enable-vulkan --enable-libdavs2 --enable-libxavs2 --enable-libuavs3d --enable-opencl --enable-opengl --enable-libopenh264 --enable-openssl --extra-cflags=-fopenmp --extra-libs=-lgomp --extra-cflags=-DLIBTWOLAME_STATIC --extra-libs=-lstdc++ --extra-cflags=-DCACA_STATIC --extra-cflags=-DMODPLUG_STATIC --extra-cflags=-DCHROMAPRINT_NODLL --extra-libs=-lstdc++ --extra-cflags=-DZMQ_STATIC --extra-libs=-lpsapi --extra-cflags=-DLIBXML_STATIC --extra-libs=-liconv --disable-w32threads --extra-cflags=-DKVZ_STATIC_LIB --enable-nonfree --extra-cflags=-DAL_LIBTYPE_STATIC --extra-cflags=-IC:/media-autobuild_suite/local64/include/AL --extra-version=g797c2ecc8f+4
ERROR: rav1e >= 0.4.0 not found using pkg-config

If you think configure made a mistake, make sure you are using the latest
version from Git.  If the latest version fails, report the problem to the
ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.freenode.net.
Include the log file "ffbuild/config.log" produced by configure as this will help
solve the problem.
configure failed. Check C:/media-autobuild_suite/build/ffmpeg-git/build-static-64bit/ab-suite.configure.log
This is required for other packages, so this script will exit.
19:17:00   Creating diagnostics file...

All relevant logs have been anonymously uploaded to https://0x0.st/-bP5.zip
Copy and paste [logs.zip](https://0x0.st/-bP5.zip) in the GitHub issue.
Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
Try running the build again at a later time.
```

My [logs.zip](https://0x0.st/-bP5.zip).

At first I thought this was related to #1910 / 3e2631f but the problem persisted after a `git revert 3e2631f`.

**Comment by 1480c1** [comment]
```bash
require_pkg_config librav1e rav1e >= 0.4.0 rav1e.h rav1e_context_new
check_pkg_config librav1e rav1e >= 0.4.0 rav1e.h rav1e_context_new
test_pkg_config librav1e rav1e >= 0.4.0 rav1e.h rav1e_context_new
pkgconf --exists --print-errors rav1e >= 0.4.0
Package dependency requirement 'rav1e >= 0.4.0' could not be satisfied.
Package 'rav1e' has version '0.3.0', required version is '>= 0.4.0'
ERROR: rav1e >= 0.4.0 not found using pkg-config
```
seems like your rav1e is not being updated

**Comment by 13steinj** [comment]
I have absolutely no idea why. I didn't change anything on the msys2 system. In fact one of these days I started completely from scratch, same issue.

But now suddenly it works?

I'll file this under "msys had some file synchronization issues, probably" since I've had such issues before with msys2. Or "ffmpeg had a change at some point that broke builds that has since been fixed" (assuming m-ab-s uses a dev/master branch and not a version tag).
